### PR TITLE
Removing figureblock2

### LINKED ISSUE #3003: Remove FigureBlock2
**Describe the bug**
It doesn't appear that `FigureBlock2` is currently in use on staging or prod and therefore, should be removed. This is a follow-up to #2999.

**To Reproduce**
1. Go to `/network-api/networkapi/wagtailpages/customblocks.py`
2. Line 106 still has unused `FigureBlock2` class.
3. Verify that it is not in use by:
- Searching the block inventory on staging and production
- Searching the codebase for the class

**Additional context**
Related PRs/Issues #2696 , #2760 , #2773 , #2782 , #1902

### PR DESCRIPTION
**Pull request by harshs14**
Fixes #3003

### CODE REVIEW

**Comment by harshs14** [comment]
@alanmoo Please review the PR

File: network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
       <div id="cta-anchor">
         <div class="mt-5 mt-md-0">
           {% cta page %}
-
+c
```
Comment (alanmoo) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html]: Remove this

File: network-api/networkapi/wagtailpages/models.py [hunk 2]
```diff
@@ -25,9 +25,7 @@
     get_page_tree_information,
     get_content_related_by_tag
 )
-
 # TODO:  https://github.com/mozilla/foundation.mozilla.org/issues/2362
-from .donation_modal import DonationModals  # noqa: F401
```
Comment (alanmoo) [inline network-api/networkapi/wagtailpages/models.py]: Why is this being removed?

Comment (harshs14) [reply network-api/networkapi/wagtailpages/models.py]: sorry, removed it by mistake

File: network-api/networkapi/wagtailpages/models.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 from django.http import HttpResponseRedirect
 from taggit.models import Tag
 
-from . import customblocks
+from networkapi.wagtailpages import customblocks
```
Comment (patjouk) [inline network-api/networkapi/wagtailpages/models.py]: Can you revert this change too? (cf https://www.python.org/dev/peps/pep-0328/)

File: Pipfile [hunk 4]
```diff
@@ -33,6 +33,7 @@ wagtail-modeltranslation = "==0.10.1"
 cloudinary = "*"
 wagtail-experiments = "*"
 pyyaml = "==5.1"
+invoke = "*"
```
Comment (patjouk) [inline Pipfile]: This is not necessary, can you remove it? You will need to lock again.

File: network-api/networkapi/wagtailpages/models.py [hunk 5]
```diff
@@ -25,7 +25,6 @@
     get_page_tree_information,
     get_content_related_by_tag
 )
-
```
Comment (patjouk) [inline network-api/networkapi/wagtailpages/models.py]: Can you bring this space back?

File: Pipfile (hunk 4 above)
Comment (patjouk) [reply Pipfile]: Actually, can you uncommit the pipfile and pipfile.lock from that PR? There's dependencies updates in it that I would like to handle myself. Thanks

Comment (harshs14) [reply Pipfile]: This got added when I fetched from upstream, let me remove it and then commit again

File: network-api/networkapi/wagtailpages/models.py (hunk 5 above)
Comment (harshs14) [reply network-api/networkapi/wagtailpages/models.py]: yes, let me make the changes

File: Pipfile (hunk 4 above)
Comment (patjouk) [reply Pipfile]: Please, remove the lock file too.

Comment (harshs14) [reply Pipfile]: ok

Comment (patjouk) [reply Pipfile]: from your PR, not from the project...

Comment (harshs14) [reply Pipfile]: working on it

Comment (harshs14) [reply Pipfile]: made changes in Pipfile.lock as it was at initial

Comment (harshs14) [reply Pipfile]: is it alright now?

**Comment by youriwims** [comment]
Ran `invoke docker-test-python` and test passed with an 🆗 . Once this lands, I'll double check staging to make sure all is good. @alanmoo @patjouk

**Comment by alanmoo** [comment]
Thanks, Youri. I'm going to let @patjouk take another look at this Monday before merging.

**Comment by patjouk** [comment]
Review app seems okay. Merging it and will do a few checks on staging.
Thanks for your contribution!
